en/02_Developer_Guides/08_Performance/07_Read_only_database_replicas.md

```diff
@@ -25,7 +25,9 @@ SS_DATABASE_USERNAME_REPLICA_01="my-replica-user"
 SS_DATABASE_PASSWORD_REPLICA_01="my-replica-password"
 ```
 
-Replicas cannot define different configuration values for `SS_DATABASE_CLASS`, `SS_DATABASE_NAME`, or `SS_DATABASE_CHOOSE_NAME`. They are restricted to prevent strange issues that could arise from having inconsistent database configurations across replicas.
+Replicas cannot define different configuration values for `SS_DATABASE_CLASS`, `SS_DATABASE_NAME`, or `SS_DATABASE_CHOOSE_NAME`. Those are restricted to prevent strange issues that could arise from having inconsistent database configurations across replicas.
```

Comment:
"They" becomes "Those"
Reading this, "They" felt like it was talking about replicas rather than about the env vars.